src/redux/books/books.js

```diff
@@ -0,0 +1,31 @@
+export const addBook = ({ name, category }) => ({
+  type: 'ADD_BOOK',
+  id: Date.now(),
+  name,
+  category,
+});
+
+export const removeBook = (id) => ({
+  type: 'REMOVE_BOOK',
+  id,
+});
+
+const books = (state = [], action) => {
```

Comment:
**[OPTIONAL] extra review comment **


Hello @harlexkhal  your project was selected as a random quality control check. ere is an additional comment that was missed by your code reviewer. 

You should use the dicks pattern for your reducers. Please take a look at the example here: https://github.com/erikras/ducks-modular-redux#example. You are missing 2 things, the second one:

- this function should be exported as default and called `reducer`.